### PR TITLE
Project builds with Maven

### PR DESCRIPTION
**Pull request by peter-budo**
- Moved project on Maven build
- Removed .classpath and .project as originally request in Pull #2

Please use IntelliJ instead of Eclipse that doesn't handle maven well.
In IntelliJ File > Open, navigate to project and select pom.xml. Watch for pop-up messages in right top corner and Enable Maven auto-import

To run project, on command line execute `mvn clean install` (recommended, IDEs are nice but...) make sure that either usb device plugged or emulator already running.

### CODE REVIEW

File: test/com/ninetwozero/battlechat/http/LoginHtmlParserTest.java [hunk 1]
```diff
@@ -1,74 +0,0 @@
```
Comment (karllindmark) [inline test/com/ninetwozero/battlechat/http/LoginHtmlParserTest.java:-1]: Did you remove this file?

**Comment by karllindmark** [comment]
Question:

Why are things in other directories? Also, four Novoda images are there - for any specific purpose? :8ball: 

Is there a hotkey for "Extract Android String..." in IntelliJ? I usually also do renaming of variables via ALT+SHIFT+R - is that possible?

File: test/com/ninetwozero/battlechat/http/LoginHtmlParserTest.java (hunk 1 above)
Comment (peter-budo) [inline test/com/ninetwozero/battlechat/http/LoginHtmlParserTest.java:-1]: No, in core module test check https://github.com/peter-budo/Battle-Chat/blob/mavenizing/core/src/test/java/com/ninetwozero/battlechat/http/LoginHtmlParserTest.java

**Comment by peter-budo** [comment]
Four Novoda images for instrumentation, they are automatically generated by archetype, we do not care for it is not part of distributed app and it can be changes any time if you really want.

Anything can be renamed with Ctrl+F6 (packages, classes, variables)

Extract android string I will have to look up

**Comment by karllindmark** [comment]
I'll have a look on Thursday due to sleep.

**Comment by karllindmark** [comment]
I don't really get the Maven structure!

**Comment by peter-budo** [comment]
I cannot explain it if you are unreachable....

**Pull request closed without merging**